core/src/test/java/org/infinispan/partitionhandling/OptimisticTxPartitionAndMergeDuringCommitTest.java

```diff
@@ -16,7 +16,7 @@
  * @author Pedro Ruivo
  * @since 8.0
  */
-@Test(groups = "functional", testName = "partitionhandling.OptimisticTxPartitionAndMergeDuringCommitTest")
+@Test(groups = "unstable", testName = "partitionhandling.OptimisticTxPartitionAndMergeDuringCommitTest", description = "ISPN-8232")
```

Comment:
Should keep the original group as well

```suggestion
@Test(groups = {"functional", "unstable"}, testName = "partitionhandling.OptimisticTxPartitionAndMergeDuringCommitTest", description = "ISPN-8232")
```

Reply:
Does that still prevent the test from being run by default ?

Reply:
Yes

Reply:
Done